i18n/es.json

```diff
@@ -2570,6 +2570,10 @@
     "id": "app.admin.test_email.failure",
     "translation": "Conexión fallida: {{.Error}}"
   },
+  {
```

Comment:
Please remove the es.json changes, only English is required, the rest is handled by translations team.